dash/dash-renderer/src/observers/requestedCallbacks.ts

```diff
@@ -125,7 +125,7 @@ const observer: IStoreObserverDefinition<IStoreState> = {
                                     exg => Boolean(exg),
                                     pluck('executionGroup', groupWithoutInitial)
                                 ).slice(-1)[0]
-                            },
+                            } as any,
```

Comment:
are we sure, we want to do this.

Reply:
You're right that we should try to minimize escapes like this, but the upgraded TypeScript barfed before this change and I couldn't find a better solution easily. IIRC the issue is `.slice(-1)[0]` could fail to yield an element, except that we know from how we constructed `groupWithoutInitial` that there will be at least one. @eff-kay perhaps at some point (after this release) you can take on the `@types/ramda` bump and also clean this up? I'm definitely not a power TS user, and it seems like Ramda pushes TS pretty hard.